### PR TITLE
Add core and dev directories

### PR DESCRIPTION
**Pull request by carmocca**
# Before submitting

Split the codebase into core utilities and development utilities

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ pip install lightning_tools
 __Usage:__
 
 ```bash
-python -m lightning_tools [group] [command]
+python -m lightning_tools.dev [group] [command]
```
Comment (Borda) [inline README.md:82]: Can we describe how this `.dev` work? :)

Comment (carmocca) [reply README.md:82]: What do you mean exactly? It's the module name

Comment (Borda) [reply README.md:82]: ok, get it :)

**Review by akihironitta — APPROVED** [review]
Sorry for the delay. LGTM!

**Comment by akihironitta** [comment]
Let me resolve the conflict now 👀

**Review by Borda — APPROVED** [review]

**Pull request merged**